about.html

```diff
@@ -0,0 +1,149 @@
+                </p>
+            </div>
+            <div class="about-sub-text">
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            </p>
+                <p>@2022, SelfDev. All rights reserved</p>
+            </div>
+        </section>
+
+        <section class="container about-section-2">
+           <div class="title">
+                <h3>Creative commons global summit 2015 logo</h3>
+                <hr class="hr">
+            </div>
+          <div class="about-section-2-text">
+              <p>
+                  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+                  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+              </p>
+          </div>
+          <figure class="about-section-2-img">
+            <img src="./images/selDev-logo.png" alt="logo">
+          </figure>
+        </section>
+
+        <section class="summits">
+            <div class="container">
+                <div class="title">
+                    <h3>See the past CC Summits</h3>
+                    <hr class="hr">
+                    <p>
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    </p>
+                </div>
+                <div class="summit-gird-row">
+                    <div class="summit-grid-col summit-bg">
+                        <div class="title-overlay">
+                            <div class="summit-title">
+                                <h3>2022</h3>
+                                <p>CC global summits in Uganda</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="summit-grid-col summit-bg">
+                        <div class="title-overlay">
+                            <div class="summit-title">
+                                <h3>2022</h3>
+                                <p>CC global summits in Uganda</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="dark-bg">
+        <div class="container footer">
+            <a href="index.html" class="footer-logo">
+                <img src="./images/selDev-logo.png" alt="logo">
+            </a>
+            <div class="footer-details">
+                <p>2022 SelfDev. All rights reserved</p>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                </p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- Good job with the footer 👌 Just a few issues to iron out here.
- [ ] In the desktop view, there is an additional white space after the dark footer. Kindly remove this please 👍 
- Your design:
![Annotation 2022-05-26 085146](https://user-images.githubusercontent.com/85113567/170449108-049af529-8cd7-4923-a524-c8ea298ca42b.jpg)

- [ ] In the mobile view, there is supposed to be a partner's section and a white footer. It would be great if you could correct that too 👍 
- Your design:
![Annotation 2022-05-26 085211](https://user-images.githubusercontent.com/85113567/170449260-fce7db02-8fb3-4bca-8cb5-05b6b1c2b32e.jpg)
- Required design:
![Annotation 2022-05-26 085230](https://user-images.githubusercontent.com/85113567/170449313-94344395-ed0b-4160-b970-e2913ce75a6f.jpg)